docs/src/getting-started/how-it-compares.md

```diff
@@ -1,42 +1,43 @@
 # How it Compares
 
-Since many users may be familiar with indexing by using a solution like The Graph, it may be helpful to provide a comparison between it and the Fuel indexer. Generally, the biggest conceptual differences between the two are as follows:
+- Since many users may be familiar with indexing by using a solution like The Graph, it may be helpful to provide a comparison between The Graph and the Fuel indexer.
 
-- The Fuel indexer does not support publishing indexers to decentralized networks. That being said, the Fuel indexer is developed with the intention of allowing anyone to run an instance and host whatever indexers they want.
-- Contracts on the Fuel network can be indexed without the need to add additional events. Whether a user wanted to index information from the very start of a contract or a recent block, the process is the same: define the schema and manifest, write custom handler logic, and deploy to an indexer service.
+## Differential Value
 
-Legend:
+- Generally, the biggest conceptual differences between other indexer services such as The Graph, and Fuel's indexer service include: indexing speed, general ease of use, and the type of data that can be indexed.
```

Comment:
Similar to the introduction, we should make this a regular sentence since it's introducing the idea of differential value.

Also, consider the following adjustment that moves the Fuel indexer to the start of the sentence and makes a few other tweaks:
> Generally, the biggest conceptual differences between Fuel's indexer service and other indexer services (such as The Graph) are indexing speed, general ease of use, and indexable data types.